boards/arm/lpcxpresso55s69/CMakeLists.txt

```diff
@@ -33,4 +36,65 @@ if (CONFIG_BUILD_WITH_TFM)
 	if(${SREC_CAT} STREQUAL SREC_CAT-NOTFOUND)
 		message(FATAL_ERROR "'srec_cat' not found. Please install it, or add it to $PATH.")
 	endif()
+
+	if (CONFIG_TFM_BL2_FALSE)
+		#merge tfm_s and zephyr_ns to a single image
+		#offset needs to be the same value as flash_layout.h in TFM
+		set_property(GLOBAL APPEND PROPERTY extra_post_build_commands
+			COMMAND ${SREC_CAT}
+			ARGS ${CMAKE_BINARY_DIR}/tfm/bin/tfm_s.bin -Binary
+				${CMAKE_BINARY_DIR}/zephyr/${KERNEL_BIN_NAME} -Binary
+				-offset 0x40000
```

Comment:
Should this not use the Kconfig setting `${CONFIG_FLASH_LOAD_OFFSET}`? 
https://github.com/zephyrproject-rtos/zephyr/blob/f70e29281cb7c396891027180aefef3e0b5a6db1/boards/arm/lpcxpresso55s69/Kconfig.defconfig#L59-L60